modules/applications/pages/package-and-deploy-your-application.adoc

```diff
@@ -1,7 +1,67 @@
 = Package and deploy your application
-:description:
+:description: Step-by-step guide to transition from development to production using Bonita UI Builder
 
 [NOTE]
 ====
 For Subscription editions only.
 ====
+
+Once you've designed an application that interacts with your xref:interact-with-your-bonita-process.adoc[bonita process], and xref:builder-declare-interface-in-bonita.adoc[declared] it in Bonita, it's time to push your interface to production
+
+
+== Prerequisites
+
+Before starting, ensure you have **Docker** the following tools installed on your system.
+
+== Move from Development to Production
+
+=== Export Your Application
+
+Once the design is complete, export the application(s) from Bonita UI Builder by clicking the `export` button on the home page. This export will be used for building the production binary.
```

Comment:
Give more details on where the "export" button is located, this is not obvious...